mixed_requests/test_mixed.py

```diff
@@ -1,96 +1,379 @@
+   r[2] = wrk.format("GET", "/")
+   r[3] = wrk.format("HEAD", "/")
+   r[4] = wrk.format("POST", "/")
+   r[5] = wrk.format("PUT", "/")
+   r[6] = wrk.format("PATCH", "/")
+   r[7] = wrk.format("DELETE", "/")
+   req = table.concat(r)
+end
+
+request = function()
+   return req
+end
+"""
+
+
+
+class MixedRequests(tester.TempestaTest):
+
+    backends = [
+        {
+            'id' : 'nginx',
+            'type' : 'nginx',
+            'check_ports' : [
+                {
+                    "ip" : "${server_ip}",
+                    "port" : "8000",
+                }
+            ],
+            'status_uri' : 'http://${server_ip}:8000/nginx_status',
+            'config' : """
+pid ${backend_pid};
+worker_processes  auto;
+
+events {
+    worker_connections   1024;
+    use epoll;
+}
+
+http {
+    keepalive_timeout ${server_keepalive_timeout};
+    keepalive_requests ${server_keepalive_requests};
+    sendfile         on;
+    tcp_nopush       on;
+    tcp_nodelay      on;
+
+    open_file_cache max=1000;
+    open_file_cache_valid 30s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors off;
+
+    # [ debug | info | notice | warn | error | crit | alert | emerg ]
+    # Fully disable log errors.
+    error_log /dev/null emerg;
+
+    # Disable access log altogether.
+    access_log off;
+
+    server {
+        listen        ${server_ip}:8000;
+
+        location / {
+            return 200;
+        }
+        location /nginx_status {
+            stub_status on;
+        }
+    }
+}
+""",
+        }
+    ]
+
+    clients = [
+        {
+            'id' : 'wrk',
+            'type' : 'wrk',
+            'addr' : "${tempesta_ip}:80",
+        },
+    ]
+
+    tempesta = {
+        'config' : """
+cache 0;
+listen 80;
+
+srv_group default {
+    server ${server_ip}:8600;
+}
+
+vhost default {
+    proxy_pass default;
+}
+""",
+    }
+
+    def routine(self, lua):
+        nginx = self.get_server('nginx')
+        wrk = self.get_client("wrk")
+
+        wrk.set_script("wrk", lua)
+
+        nginx.start()
+        nginx.wait_for_connections(timeout=3)
+
+        self.start_tempesta()
+
+        wrk.start()
+        self.wait_while_busy(wrk)
+        wrk.stop()
+        return wrk
+
+    def test_pipeline(self):
+        self.routine(pipeline_lua)
+
+    def test_real(self):
+        self.routine(lua_real)
+
+    def test_real2(self):
+        self.routine(lua_real2)
+
+    def test_real_pipelined(self):
+        self.routine(lua_real_pipelined)
+
+    def test_get_post(self):
+        self.routine(lua_get_post)
+
+    def test_head_get(self):
+        self.routine(lua_head_get)
+
+    def test_post_empty(self):
+        self.routine(lua_post_empty)
+
+    def test_post_small(self):
+        self.routine(lua_post_small)
+
+    def test_post_big(self):
+        # Too big text to put it here explicitly
+
+        text = "content " * 8192
+        lua_post_big = r"""local body = [[
+""" + text + r"""]]
```

Comment:
Such multiline strings breaks indentation and makes code hard to read. I wonder whether there any good python practice for such cases. Any ideas?